lively.collab/comments/commentBrowser.js

```diff
@@ -270,6 +293,14 @@ export class CommentBrowser extends Window {
     group.remove();
   }
 
+  async applyResolveStatus (comment, referenceMorph) {
+    await this.removeCommentForMorph(comment, referenceMorph);
+    await this.addCommentForMorph(comment, referenceMorph);
+  }
```

Comment:
Maybe we could either put it into the `browser mode / archive` section or make a section `resolve comments`?